### DISCUSSION TITLE
FLAC not fully converting/ripping certain songs while above 96khz signal processing setting during rip/conversion

### DISCUSSION THREAD

**Discussion by impotentfield** [enzo1982/freac#512] [Audio Format Discussion]
i have a certain song that isnt fully ripping during rip and signal processing setting while using flac. anything above 96khz convesiong while ripping flac with custom options. i need help with this or else just simply posting to address a bug....

**Comment by enzo1982** [comment]
You're writing _ripping/converting_, but as audio CDs are always sampled at 44.1 kHz and you're writing specifically about 96 kHz, I assume you mean _converting_ existing 96 kHz files only.

In that case, can you please let me know the exact settings to reproduce the issue and make the song available to me at support@freac.org? I will then analyze the issue and try to find a solution.

**Comment by impotentfield** [comment]
i wrote an email to support@freac.org about the issue i am having which i can live with but it remains that the bug could be substancial in program operation. i do like high quality conversions and i experiment alot with freac. thank you.
